validator/accounts/v2/wallet_create_test.go

```diff
@@ -217,6 +217,38 @@ func TestCreateWallet_Derived(t *testing.T) {
 	assert.DeepEqual(t, derived.DefaultKeymanagerOpts(), cfg)
 }
 
+// TestCreateWallet_IfExistingWallet checks for expected error if trying to create a wallet where they is already one.
+func TestCreateWallet_IfExistingWallet(t *testing.T) {
```

Comment:
`TestCreateWallet_WalletAlreadyExists` would be a name that is more in line with the rest of the tests in the project.